services/Directory/data/FilterListViewUrl.json

```diff
@@ -16480,5 +16480,11 @@
     "id": 2936,
     "primariness": 1,
     "url": "https://raw.githubusercontent.com/jarelllama/Scam-Blocklist/main/adblock.txt"
+  },
+  {
+    "filterListId": 2519,
+    "id": 2923,
```

Comment:
```suggestion
    "id": 2937,
```

Reply:
each `id` in this file needs to be unique. we already have an `id` of `2923` which is why the build is complaining.